docs/_docs/2_2_writing_data.md

```diff
@@ -210,6 +210,8 @@ Sample config files for table wise overridden properties can be found under `hud
   --op BULK_INSERT
 ```
 
+For detailed information on how to configure and use `HoodieMultiTableDeltaStreamer`, please refer blog section.
```

Comment:
It would be better to add a link to the address of the blog. WDYT?

Reply:
Looks like I need to generate index.html file to be able to add the link. @nsivabalan Can you please guide me here?